docker/Dockerfile

```diff
@@ -124,16 +126,16 @@ RUN cd /opt && \
 
 # jemalloc
 RUN cd /opt && \
-    wget https://github.com/jemalloc/jemalloc/archive/3.6.0.tar.gz && \
-    tar -zxvf 3.6.0.tar.gz && \
-    (cd jemalloc-3.6.0 && \
+    wget https://github.com/jemalloc/jemalloc/archive/4.5.0.tar.gz && \
+    tar -zxvf 4.5.0.tar.gz && \
```

Comment:
This is for jemalloc, I just ported over the version being used in the cosmos Dockerfile, since that has been updated more recently than the one for Rocksplicator